### ISSUE TITLE
Train dataset problem

### ISSUE THREAD

**Issue by oskmilke** [QijinXu/APLRL#2]
thank you for your work, In the paper you provided, it mentions that the Places2 training set uses 12 scenes and 1,200 test images. Could you please let me know which 12 scenes were used?

**Comment by QijinXu** [comment]
Of course，here are the scenes：alcove arch bedchamber butte chalet cliff field_cultivated field_road house islet synagogue_outdoor valley.



---- Replied Message ----
| From | ***@***.***> |
| Date | 07/14/2025 16:53 |
| To | ***@***.***> |
| Cc | ***@***.***> |
| Subject | [QijinXu/APLRL] Train dataset problem (Issue #2) |
oskmilke created an issue (QijinXu/APLRL#2)

thank you for your work, In the paper you provided, it mentions that the Places2 training set uses 12 scenes and 1,200 test images. Could you please let me know which 12 scenes were used?

**Comment by oskmilke** [comment]
Thanks

**Comment by oskmilke** [comment]
According to your paper, the test set consists of 1,200 images. I would appreciate it if you could share how these images were selected or constructed.

**Comment by QijinXu** [comment]
The images in the test set are completely randomly selected



---- Replied Message ----
| From | ***@***.***> |
| Date | 07/15/2025 18:34 |
| To | ***@***.***> |
| Cc | Qijin ***@***.***>***@***.***> |
| Subject | Re: [QijinXu/APLRL] Train dataset problem (Issue #2) |
oskmilke left a comment (QijinXu/APLRL#2)

According to your paper, the test set consists of 1,200 images. I would appreciate it if you could share how these images were selected or constructed.

**Comment by oskmilke** [comment]
Thank you very much for taking the time to reply.
